testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/AbstractHttpClientTest.java

```diff
@@ -1119,6 +1120,10 @@ SpanDataAssert assertClientSpan(
               if (httpClientAttributes.contains(UrlAttributes.URL_FULL)) {
                 assertThat(attrs).containsEntry(UrlAttributes.URL_FULL, uri.toString());
               }
+              if (httpClientAttributes.contains(UrlIncubatingAttributes.URL_TEMPLATE)) {
```

Comment:
might be nice to have a boolean option for tests to say they're emitting this so we can assert more strongly here